labgrid/driver/sshdriver.py

```diff
@@ -62,7 +62,7 @@ def _start_own_master(self):
         env = os.environ.copy()
         if self.networkservice.password:
             env['SSHPASS'] = self.networkservice.password
-        self.process = subprocess.Popen(args, env=env)
+        self.process = subprocess.Popen(args, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Why do you change to PIPEs here?

Reply:
ups - leftover from testing. prevents ssh from messing up my terminal in case something goes wrong but it might more interesting to get the output back now- i will revert it. just need to make sure its still working without before ...